android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java

```diff
@@ -0,0 +1,107 @@
+package com.android.i18n.addressinput.autocomplete.gmscore;
```

Comment:
The corresponding unit test has the license....

But indeed if it is some of the comments look like they need sanitising.